index.html

```diff
@@ -9,6 +9,17 @@
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <nav class="nav-section">
+    <div class="name">
+      <a href="#">Francis Ugorji</a>
+    </div>
+    <ul>
+      <li><a class="nav-menu" href="#">Portfolio</a></li>
+      <li><a class="nav-menu" href="#">About</a></li>
+      <li><a class="nav-menu" href="#">Contact</a></li>
+      <li><a href="#"><img src="Images/Mailbox-Icon.png" alt="A mailbox icon"></a></li>
```

Comment:
- [ ] [OPTIONAL] To make the transition using these links, add inside the `href` attribute the value of id for every respective id section. For instance:
```
<a href="#section-home">Home</a> // Link to go to section-home section by id.

<section id="section-home"> 
</section>
```

Reply:
Well solved